BackEnd/src/main/java/com/exadel/carpoolfree/model/PassengerDrive.java

```diff
@@ -0,0 +1,34 @@
+package com.exadel.carpoolfree.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Column;
+import javax.persistence.EmbeddedId;
+import javax.persistence.Entity;
+import javax.persistence.ManyToOne;
+import javax.persistence.MapsId;
+import javax.persistence.Table;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "passenger_drive")
+public class PassengerDrive {
+
```

Comment:
I don't think adding an entity without direct business meaning is a good idea. We should consider using the same entities that we already have (User and Drive) and have a join table between them. This join table exists only in DB and it doesn't need a corresponding Java representation. 

Have a look https://www.baeldung.com/hibernate-one-to-many

At the end we should only have `Drive` and `User` classes and `passender_drive` join table

Reply:
I added this entity because we have additional column "start_coordinate_of_passengers" in table "PassengersDrive"

Reply:
missed that, good point!